main/design_documents/pacbti-tfm.rst

```diff
@@ -67,6 +67,27 @@ Example:
   #endif
   }
 
-A real-world use-case of these macros is in the stack-unwinding runtime for
-exceptions where the runtime unwinding is potentially better with ``AUTG``
-rather than using ``AUT`` which forces register pressure.
+
+Example (AArch64):
+--------
+
+::
+
+  #if __ARM_FEATURE_PAC_DEFAULT
+    // Authenticated return required.
+  # if __ARM_FEATURE_PAUTH
+    // Have PAUTH instructions
+    asm ("retaa");
+  # else
+    // Use authentication from NOP space.
+    asm ("autiasp;ret");
+  # endif
+  #else
+    // No authentication required.
+    asm ("ret");
+  #endif
+
+A more practical example of these macros is in the stack-unwinding
+runtime for exceptions. In that case the runtime unwinding is improved
+by using ``AUTG`` rather than ``AUT`` because it frees up an
+additional register.
```

Comment:
I think this is referred to the AArch32 example? If so, should we move it up, right after the AArch32 example? In any case, for someone like me who is not an expert of this feature, it doesn't seem to add anything to the understanding of the mechanism. Would it be possible to add an example that shows the way these macro should be use for stack unwinding?